src/media/css/app-list.styl

```diff
@@ -117,9 +117,19 @@ section.langpacks > div {
         &:nth-child(2n) {
             border-right: 0;
             padding-right: 0;
+
+            +rtl() {
+                border-right: $border;
+                padding: 20px 50px 20px 0;
```

Comment:
The issue here though is that we're cascading a rule that we don't want to cascade. I'm all for using the cascade but I don't see the purpose in overriding the default with the LTR case to then revert the LTR change _and_ reverse it for RTL.

I see no benefit to cascading the LTR case here and I don't know what problems we'll run into by separating the LTR and RTL styles explicitly.

That being said if you want to flex this and use `space-between` the padding issue goes away. Still have to deal with the border though so it may end up the same.